### PR TITLE
Enable magnum tests for SOC7

### PR DESCRIPTION
**Pull request by gosipyan**
Enabled magnum tempest filter for SOC7
Deploy magnum with ssl enabled

### CODE REVIEW

**Comment by gosipyan** [comment]
testing https://ci.suse.de/job/cloud-crowbar7-job-gate-ha-deploy-x86_64/286/

**Comment by gosipyan** [comment]
One magnum test failing due to missing backport abae955fdf8f3775e0b4d695fd9e9c803e667d4f

**Comment by gosipyan** [comment]
testing with SOC8 https://ci.suse.de/job/cloud-crowbar8-job-gate-ha-deploy-x86_64/292/

File: jenkins/ci.suse.de/cloud-maintenance.yaml [hunk 1]
```diff
@@ -60,18 +60,16 @@
           cloudsource: GM7+up
           ses_enabled: false
           update_after_deploy: false
-          # magnum not fully passing yet
           tempest_filter_list: "\
             keystone,swift,glance,cinder,neutron,nova,ceilometer,fwaas,\
-            trove,aodh,heat,manila,lbaas"
+            trove,aodh,heat,magnum,manila,lbaas"
```
Comment (flaviodsr) [inline jenkins/ci.suse.de/cloud-maintenance.yaml:65]: I would recommend not enabling those tests here until we have released those changes or they will keep breaking the maintenance-gating job.

Comment (gosipyan) [reply jenkins/ci.suse.de/cloud-maintenance.yaml:65]: @flaviodsr you think I can remove maintenance part and get this merged ? and later add part for maintenance jobs

**Pull request merged**